tft_bot/helpers/screen_helpers.py

```diff
@@ -230,3 +231,33 @@ def get_on_screen_multiple_any(window_title: str, paths: list[str], precision: f
             return True
 
     return False
+
+
+_TESSERACT_CONFIG = '--oem 3 --psm 7 -c tessedit_char_whitelist=0123456789 -c page_separator=""'
+
+
+def get_gold() -> int:
```

Comment:
Since this is using tesseract, should this not be in the tesseract event module? 🤔

Reply:
I moved it here because, technically, it's screen reading, and that also enables us to re-use it in future ocr economy modes, but I am fine with moving it back if you think that makes more sense :)

Reply:
Gotcha! Yeah to be honest that's fine with me for it to be there -- the hangup I have is that it seems weird for the tesseract-specific functions to be mixed in with the regular ones 🤔